blob/azureblob/example_test.go

```diff
@@ -109,3 +111,56 @@ func Example_openBucketFromURL() {
 	}
 	defer bucket.Close()
 }
+
+func ExampleOpenBucket_usingAADCredentials() {
+	const (
+		// Your Azure Storage Account.
+		accountName = azureblob.AccountName("my-account")
+
+		// Your Azure AAD Service Principal with access to the storage account
```

Comment:
Nit: Please end comments with a period, here and below.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences.